plugin/pkg/admission/securitycontext/scdeny/admission.go

```diff
@@ -53,42 +53,53 @@ func (p *plugin) Admit(a admission.Attributes) (err error) {
 		return apierrors.NewBadRequest("Resource was marked with kind Pod but was unable to be converted")
 	}
 
-	if pod.Spec.SecurityContext != nil && pod.Spec.SecurityContext.SupplementalGroups != nil {
-		return apierrors.NewForbidden(a.GetResource().GroupResource(), pod.Name, fmt.Errorf("SecurityContext.SupplementalGroups is forbidden"))
+	if err = checkPodSecurityContext(pod.Spec.SecurityContext); err != nil {
```

Comment:
Could you use `err :=` form instead of assigning a value to a named return value?

In this case you need to modify function signature (`Admit(a admission.Attributes) (err error)` -> `Admit(a admission.Attributes) error`)